doc/source/user_guide/timeseries.rst

```diff
@@ -2265,6 +2265,24 @@ you can use the ``tz_convert`` method.
     Instead, the datetime needs to be localized using the ``localize`` method
     on the ``pytz`` time zone object.
 
+.. warning::
+
+    If you are using dates beyond 2038-01-18, due to current deficiencies
+    in the underlying libraries caused by the year 2038 problem, daylight saving time (DST) adjustments
+    to timezone aware dates will not be applied. If and when the underlying libraries are fixed,
+    the DST transitions will be applied. It should be noted though, that time zone data for far future time zones
+    are likely to be inaccurate, as they are simple extrapolations of the current set of (regularly revised) rules.
+
+    For example, for two dates that are in British Summer Time (and so would normally be GMT+1), both the following asserts evaluate as true:
+
+    .. ipython:: python
+
+       d_2037 = '2037-03-31T010101'
+       d_2038 = '2038-03-31T010101'
+       DST = 'Europe/London'
+       assert pd.Timestamp(d_2037, tz=DST) != pd.Timestamp(d_2037, tz='GMT')
```

Comment:
This does not seem right to me.

If you're going to keep the example (and I'm not convinced you should, *especially* since it's right around when a DST transition happens - if anything you should move it deep into summer to guarantee that the fluctuations aren't just due to the transition moving around), it would be better to make assertions about the thing you care about.

```python
assert pd.Timestamp(d_2037, tz=LON).tzname() != "GMT"
assert pd.Timestamp(d_2038, tz=LON).tzname() != "GMT"
```

Even better, though, would be a repr:

```python
>>> pd.Timestamp(d_2037, tz=LON).tzname()
'BST'
>>> pd.Timestamp(d_2037, tz=LON).tzname()
'GMT'
```

Reply:
I agree summer date would be clearer , no so sure whether your example is clearer , or whether using BST (local zone name for DST) is clearer for a global audience ... hmmm

Reply:
It doesn't really matter what offsets you use there, pick anything. The important thing is that it's obvious that they are *different* on the same date in different years, and that it's obvious that that's not due to fluctuations one way or the other in the date of the DST change.

Reply:
Is it OK to submit further changes once the PR has been approved?

Reply:
Yes you can submit further changes. I think @pganssle's suggestion about using the repr would be clearer.